core/runtime/v2/shim.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/containerd/containerd/v2/pkg/atomicfile"
 	"github.com/containerd/containerd/v2/pkg/dialer"
 	"github.com/containerd/ttrpc"
```

Comment:
nit: Are the imports properly ordered in this file.  Not related to this change, but seems like the above packages should be part of the 3rd group